packages/core/src/amazonq/lsp/lspController.ts

```diff
@@ -358,13 +371,14 @@ export class LspController {
                 result: 'Failed',
                 amazonqIndexFileCount: 0,
                 amazonqIndexFileSizeInMB: 0,
+                reason: `${e}`,
```

Comment:
```
/** Reason code or name for an event (when result=Succeeded) or error (when result=Failed). Unlike the `reasonDesc` field, this should be a stable/predictable name for a class of events or errors (typically the exception name, e.g. FileIOException). */
```
Use reasonDesc if you want to publish the contents of the exception. Otherwise try to pull `code`, `name`, or similar field out of the exception if possible. This can be done automatically if you use `telemetry.run()` instead of `emit`.

Reply:
fixed in latest commit.